cmd/lncli/cmd_open_channel.go

```diff
@@ -331,7 +343,11 @@ func openChannel(ctx *cli.Context) error {
 	case ctx.IsSet("local_amt"):
 		req.LocalFundingAmount = int64(ctx.Int("local_amt"))
 	case args.Present():
-		req.LocalFundingAmount, err = strconv.ParseInt(args.First(), 10, 64)
+		base := 10
+		bitSize := 64
+		req.LocalFundingAmount, err = strconv.ParseInt(
```

Comment:
IMO, it is less readable
Why creating new variable for the base and bitSize (instead of:

```
strconv.ParseInt(
    args.First(), 10, 64
)
```
)

Reply:
Thanks for your review! `base` and `bitSize` was introduced to satisfy the linter which flagged the previous version which I agree was more readable.

For the next reviewer, should we add a lint exception here?

Reply:
so why not using what I've wrote in my first message :D 

```
req.LocalFundingAmount, err = strconv.ParseInt(
			args.First(), 10, 64,
		)
```

Reply:
Sure, if I remember correctly I had it like you suggest but it gave me linter errors. Just checked again and the linter error wasn't showing locally. So I changed it to your suggestion.

Reply:
Looks like it works!